doc/devel/architecture.md

```diff
@@ -67,18 +42,6 @@ device information to create the definitive Device object that will be stored in
 Particular IoT Agents _may_ support autoregistration of devices into configurations, if enough information is given from
 the Southbound.
 
-#### Configurations and subservices
```

Comment:
This part has gone to some other .md? Or has been deleted as it is not useful or already described?

Reply:
As noted here: https://github.com/telefonicaid/iotagent-node-lib/pull/1470#discussion_r1338888213
and described here: https://github.com/telefonicaid/iotagent-node-lib/issues/1469
It is not valid any more